iidm/iidm-serde/src/main/java/com/powsybl/iidm/serde/ConnectableSerDeUtil.java

```diff
@@ -164,6 +209,27 @@ public static void readOptionalPQ(Integer index, Terminal t, TreeDataReader read
                 .ifPresent(t::setQ);
     }
 
+    private static String indexToString(DcTerminal dcTerminal) {
```

Comment:
Can you rename this method? The other `indexToString(...)` method takes an index as a parameter, so its name makes sense, but this is not the case here.